src/aurora/lua/scriptman.h

```diff
@@ -0,0 +1,118 @@
+/* xoreos - A reimplementation of BioWare's Aurora engine
+ *
+ * xoreos is the legal property of its developers, whose names
+ * can be found in the AUTHORS file distributed with this source
+ * distribution.
+ *
+ * xoreos is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 3
+ * of the License, or (at your option) any later version.
+ *
+ * xoreos is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with xoreos. If not, see <http://www.gnu.org/licenses/>.
+ */
+
+/** @file
+ *  Lua script manager.
+ */
+
+#ifndef ENGINES_AURORA_LUA_SCRIPTMAN_H
+#define ENGINES_AURORA_LUA_SCRIPTMAN_H
+
+extern "C" {
+
+#include "lua/lua.h"
+
+}
+
+#include "src/common/singleton.h"
+
+namespace Common {
+	class UString;
+}
+
+namespace Aurora {
+
+namespace Lua {
+
+/** Lua script manager. */
+class ScriptManager : public Common::Singleton<ScriptManager> {
+public:
+	ScriptManager();
+	~ScriptManager();
+
+	/** Initialize the script subsystem. */
+	void init();
+	/** Deinitialize the script subsystem. */
+	void deinit();
+
+	/** Was the script subsystem successfully initialized? */
+	bool ready() const;
+
+	/** Execute a sctipt file. */
```

Comment:
Ah, you fixed the typo in a908f9b. If you could instead move the fix directly here (by using an interactive git rebase), i.e. have it look like you never made the typo at all, that would be better.